indexer/packages/v4-protos/src/codegen/dydxprotocol/clob/query.ts

```diff
@@ -259,15 +306,60 @@ export interface StreamOrderbookUpdatesResponseSDKType {
    */
 
   snapshot: boolean;
+}
+/** StreamOrderbookUpdatesOrderFills contains updates for orderbook fills. */
+
+export interface StreamOrderbookUpdatesOrderFills {
```

Comment:
Accepting name suggestions.

Reply:
maybe drop StreamOrderbookUpdates or something? kind of verbose